packages/terra-filter-view/src/_Frame.jsx

```diff
@@ -0,0 +1,540 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import Button from 'terra-button';
+import IconSearch from 'terra-icon/lib/icon/IconSearch';
+import IconIncomplete from 'terra-icon/lib/icon/IconIncomplete';
+import { KeyCodes, Variants } from './_constants';
+import Dropdown from './_Dropdown';
+import Util from './_FrameUtil';
+import styles from './_Frame.module.scss';
+
+const cx = classNames.bind(styles);
+
+const Icon = <IconSearch />;
+
+const propTypes = {
+  /**
+   * Whether the select is disabled.
+   */
+  disabled: PropTypes.bool,
+  /**
+   * When true, will disable the auto-search.
+   */
+  disableAutoSearch: PropTypes.bool,
```

Comment:
nit: This is a preference, but I think documentation should read consistently for PropTypes, in this case specifically booleans. I would choose between `Whether the filter...` and `When true, will...` for each boolean to consistently document descriptions.